README.md

```diff
@@ -8,39 +16,124 @@
+- If you're planning to contribute BoxBilling's development, and wanting to make pull requests in the future, please directly **[run BoxBilling from the source code](#running-from-the-source-code)** instead.
 
-## Getting started
+### Obtaining a copy from GitHub releases
+We make new releases of BoxBilling whenever we have some new cool stuff to introduce you to, or when we fix some bugs 🐞. If you're planning to use BoxBilling in a production environment, [downloading BoxBilling from the source code](#running-from-the-source-code) and using it may not be a good choice. Going with releases instead of the master branch will likely be better as releases are intended to be more secure and stable. However, if you're [planning to contribute](#contributing), you will likely want to go with the [source code](#running-from-the-source-code) instead.
 
-Follow [getting started instructions](https://docs.boxbilling.com)
+First, you should [download the latest release](https://github.com/boxbilling/boxbilling/releases/latest) from our GitHub repository. Our releases typically have a file called "BoxBilling.zip" attached to them, and that's exactly what you need to download. Unlike the source code itself, releases include Composer packages, so you won't need to run Composer to install PHP packages. If you're going to use BoxBilling in a shared hosting environment, you'll most likely not have an option to access Composer, so obtaining a copy from GitHub releases would be your best option.
 
-## Download
+After you downloaded the release, unzip it somewhere, and copy the content of the ZIP archive to your web server's public folder (usually, that's called **"htdocs"** or **"public_html"**). Your web directory's structure should now look like this:
 
-[![Download Latest](http://i.imgur.com/djy4ExU.png)](https://github.com/boxbilling/boxbilling/releases/latest) 
+- htdocs
+    - bb-data
+    - bb-library
+    - bb-module
+    - **...**
 
-## Contribute
+Now, as you have everything ready to start the installation of BoxBilling, head over to your web server, and follow on-screen instructions to complete the installation using web installer. Ta-da, you've done it! 🎉
 
-Follow [contributors' guidelines](https://github.com/boxbilling/boxbilling/blob/master/CONTRIBUTING.md)
+### Running from the source code
+To clone and run BoxBilling, you'll first need to [download an up-to-date ZIP archive](https://github.com/boxbilling/boxbilling/archive/master.zip) and save it to your computer locally.
 
-## Running with Docker
+Then, extract the contents of the **"src"** folder inside the ZIP archive to your web server's public folder (usually, that's called **"htdocs"** or **"public_html"**). Your web directory's structure should now look like this:
 
-Assuming you already have [Docker](https://docs.docker.com/get-docker/) and [GNU make](https://www.gnu.org/software/make/) installed,
+- htdocs
+    - bb-data
```

Comment:
@evrifaessa I would try and leave all comments regarding folder directories and structures generalized. The majority of people wanting to install this software already understand where their root directory structure is and how it functions in their environment. I would just make a comment of such place the files from the extracted zip file in the location of your websites root directory.  You cannot write to everyone's environment for example I would be lost by this because my environment has neither it either uses /home/public_html, /var/www/html, or /var/www/httpdocs and in some instances when using multiple docker environments or a hub like I do the root is based on a structure as such /var/lib/asdfjkl;aslkdfja;lsdkfj;alsfj;sjfsd/dockername/httpdocs as sundry.

Reply:
I don't think this would confuse anyone in any way.

Reply:
Sounds good.